### PR TITLE
Document Route/Ingress accessability in restricted netowrk environments

### PR DESCRIPTION
**Pull request by tomgeorge**
### What does this PR do?
Documents the requirement of traffic from outside the cluster to be able to reach Route/Ingress hostnames

### What issues does this PR fix or reference?
https://github.com/eclipse/che/issues/15308
https://github.com/eclipse/che/issues/15187

### CODE REVIEW

File: src/main/pages/che-7/installation-guide/assembly_installing-che-in-restricted-environment.adoc [hunk 1]
```diff
@@ -29,6 +29,11 @@ To deploy {prod-short} in an environment where these external resources are not
 * Configure {prod-short} to use these images.
 
 
+== Prerequesite Notes on Network Connectivity
```
Comment (rkratky) [inline src/main/pages/che-7/installation-guide/assembly_installing-che-in-restricted-environment.adoc]:
```suggestion
== Notes on network connectivity in restricted environments
```

File: src/main/pages/che-7/installation-guide/assembly_installing-che-in-restricted-environment.adoc [hunk 2]
```diff
@@ -29,6 +29,11 @@ To deploy {prod-short} in an environment where these external resources are not
...
+
+Restricted network environments can range from a private subnet in a cloud provider, to a separate network owned by a company, cut off from the public internet.  While your organization's network will be different, Che will work *provided that the Ingress/Routes that are created for the Che components (che-server, keycloak, devfile-registry, plugin-registry) are accessible from inside the Kubernetes cluster).*  You should take into account the network topology of your environment to determine how best to accomplish this.  For example, in a network owned by a company or organization, the network administrators should ensure that traffic bound from the cluster can be routed to Ingress/Route hostnames.  In other cases, e.g. in AWS, you may need to create a proxy configuration allowing the traffic to leave the node to get to an external-facing Load Balancer.
```
Comment (rkratky) [inline src/main/pages/che-7/installation-guide/assembly_installing-che-in-restricted-environment.adoc]:
```suggestion
Restricted network environments range from a private subnet in a cloud provider to a separate network owned by a company, disconnected from the public Internet. Regardless of the network configuration, {prod-short} works *provided that the Ingress and Routes that are created for {prod-short} components ({prod-id-short}-server, keycloak, devfile-registry, plugin-registry) are accessible from inside the Kubernetes or OpenShift cluster).*

Take into account the network topology of the environment to determine how best to accomplish this. For example, on a network owned by a company or an organization, the network administrators must ensure that traffic bound from the cluster can be routed to Ingress and Route hostnames. In other cases, for example, on AWS, create a proxy configuration allowing the traffic to leave the node to reach an external-facing Load Balancer.
```

**Review by rkratky — CHANGES_REQUESTED** [review]
Thanks, @tomgeorge. I left a couple of suggestions.

File: src/main/pages/che-7/installation-guide/assembly_installing-che-in-restricted-environment.adoc (hunk 2 above)
Comment (tomgeorge) [reply src/main/pages/che-7/installation-guide/assembly_installing-che-in-restricted-environment.adoc]: I have updated based on your suggestions

**Review by ibuziuk — APPROVED** [review]

**Review by rkratky — APPROVED** [review]

**Pull request merged**